src/domain/cards/customerProfileCard/organizationCustomerDetails/OrganizationCustomerDetails.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Section from '../../../../common/section/Section';
+import LabelValuePair from '../../../../common/labelValuePair/LabelValuePair';
+import InternalLink from '../../../../common/internalLink/InternalLink';
+import { CUSTOMER_GROUP } from '../../../types';
+import { InvoicingType } from '../../../../@types/__generated__/globalTypes';
+
+export interface OrganizationCustomerDetailsProps {
+  comment?: string | null;
+  customerId?: string;
+  customerGroup: CUSTOMER_GROUP;
+  firstName: string;
+  invoicingType?: InvoicingType;
+  lastName: string;
+  organization: {
+    address: string;
+    businessId: string;
+    city: string;
+    name: string;
+    postalCode: string;
+  };
+  primaryAddress?: {
+    address: string;
+    postalCode: string;
+    city: string;
+  } | null;
+  primaryEmail?: string | null;
+  primaryPhone?: string | null;
+  showCustomerNameAsLink?: boolean;
+  ssn: string;
+}
+
+const OrganizationCustomerDetails: FunctionComponent<OrganizationCustomerDetailsProps> = ({
+  comment,
+  customerId,
+  customerGroup,
+  firstName,
+  invoicingType,
+  lastName,
+  organization,
+  primaryEmail,
+  primaryPhone,
+  showCustomerNameAsLink = false,
+  ssn,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Section title={t('customerProfile.organization').toUpperCase()}>
+        <LabelValuePair
+          label={t('customerProfile.name')}
+          value={organization.name}
+        />{' '}
+        <LabelValuePair
+          label={t('customerProfile.businessId')}
+          value={organization.businessId}
+        />
+      </Section>
+      <Section>
+        <LabelValuePair
+          label={t('customerProfile.address')}
+          value={organization.address}
+        />
+        <LabelValuePair
+          label={t('customerProfile.postalCode')}
+          value={organization.postalCode}
+        />
+        <LabelValuePair
+          label={t('customerProfile.city')}
+          value={organization.city}
+        />
+      </Section>
+      <Section>
+        <LabelValuePair
+          label={t('customerProfile.customerGroup')}
+          value={t([`common.customerGroups.${customerGroup}`])}
```

Comment:
If we changed the customer groups to the auto-generated value, we could add here a second key to the array as a [fallback translation](https://www.i18next.com/overview/api#t) if the first one doesn't resolve.